apps/dsb-message-broker/src/channel/dto/create-channel.dto.ts

```diff
@@ -1,52 +1,75 @@
 import { IsNotEmpty, IsString, IsOptional } from 'class-validator';
 import { ApiProperty } from '@nestjs/swagger';
+import { JSONSchemaType } from 'ajv';
 
 export class CreateChannelDto {
     @ApiProperty({
-        type: String,
-        description: 'Fully qualified channel name (fcqn)',
-        example: 'test.channels.testapp.apps.testorganization.iam.ewc'
```

Comment:
I think we should leave this in

Reply:
just replaced the example with format field